ansible/roles/ps4-controller/templates/ds4drv-service.j2

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=ds4drv daemon
+Requires=bluetooth.service
+After=bluetooth.service
+
+[Service]
+EnvironmentFile=/etc/donkeycar/ds4drv.env
```

Comment:
This file isn't created.

It should contain the following content : 
```
CONTROLLER_LED_COLOR=33FF42
```

You can use your variable here if you want.